dashboard/dashapi/dashapi.go

```diff
@@ -285,6 +285,7 @@ type Crash struct {
 	Report      []byte
 	MachineInfo []byte
 	Assets      []NewAsset
+	GuiltyFiles []string
```

Comment:
Why do we need to export GuiltyFiles here? How will reportings  use it?

Reply:
In this PR, it's used to determine whether the bug happened in the `fs/*.c`. It's taken by `pkg/subsystem/extract`.

Reply:
Following https://en.wikipedia.org/wiki/Presumption_of_innocence I'd recommend "SuspectedFiles"! )